examples/StandardFirmataWiFi/StandardFirmataWiFi.ino

```diff
@@ -830,6 +830,28 @@ void systemResetCallback()
   isResetting = false;
 }
 
+/*
+ * Called when a TCP connection is either connected or disconnected.
+ * TODO - figure out why the callback is not being called when using ESP8266 as a TCP server and
+ * why only connect is called when using ESP8266 as a TCP client. In both cases the actual
+ * connection is working but not reported via the callback.
+ */
+void hostConnectionCallback(byte state)
+{
+  switch (state) {
+    case HOST_CONNECTION_CONNECTED:
+      DEBUG_PRINTLN( "TCP connection established" );
```

Comment:
TODO:
- Report connected or reconnected state to host 
- Report current state (per https://github.com/firmata/protocol/issues/59) if reconnect